integration-tests/bot/chainrpc.py

```diff
@@ -160,7 +160,7 @@ def create(self, name=DEFAULT_WALLET, type='Basic', passphrase=None):
         :param name: Name of the wallet. [defualt: Default]
         :param type: Type of the wallet. [Basic|HD] [default: Basic]
         '''
-        return self.client.call('wallet_create', [name, passphrase or get_passphrase()], type)
+        return self.client.call('wallet_create', [name, passphrase or get_passphrase()], type,24)
```

Comment:
is the argument necessary? I thought it'd automatically take 24 as default?

Reply:
i'll process with default value